### DISCUSSION TITLE
adding a docs/examples directory

### DISCUSSION THREAD

**Discussion by Galz648** [pulumi/pulumi#11566] [Ideas]
there doesn't seem to be an examples directory, is there a reason for it?
I would like to be able to have one source of sample project to play around with.

**Comment by nimbinatus** [comment]
Does this help? https://github.com/pulumi/examples

**Reply by Galz648** [reply]
Yes it does! Thanks! I wonder if I didn't look enough or that there should be a straightforward reference to it on the main repository page.

Also, is there a specific reason that it's not in an 'examples' directory?

if a pull request is needed to do that, i'd be glad to contribute :)

**Reply by susanev** [reply]
hi @Galz648 i think we link to it in the pulumi/pulumi readme, but if you see a way to make it more clear and easy to access we welcome a pr!!

**Reply by cnunciato** [reply]
>I wonder if I didn't look enough or that there should be a straightforward reference to it on the main repository page.

You're right, that link is easy to miss. +1 to what @susanev said -- a PR would be awesome!

**Reply by Galz648** [reply]
Will do!

**Reply by Galz648** [reply]
my solution will consist of a submodule to the https://github.com/pulumi/examples repository, and a githook to pull the changes from the submodule 

will this be an acceptable solution? what would be an alternative? Thanks! 
@susanev @nimbinatus
